build.gradle

```diff
@@ -151,7 +151,7 @@ subprojects {
         ignoreFailures = Boolean.getBoolean("ignoreTestFailures")
         include "**/*Tests.*"
         exclude "**/*Abstract*.*"
-        classpath = it.classpath + files("${projectDir}/src/main/resources") + files("${projectDir}/src/test/resources")
+        classpath = it.classpath + files("${projectDir}/src/main/resources") + files("${projectDir}/src/test/resources") + files("${projectDir}/../../webapp/resources")
```

Comment:
Help me out here please: why this? Does this have something to do with failing tests as you mentioned? Why should the root-level project know about the webapp resources? If this is needed, couldn't you mod the classpath at the webapp project level and augment it there?